### PR TITLE
Implement Sumsub support

### PR DESCRIPTION
**Pull request by chaotixkilla**

### CODE REVIEW

File: new-lamassu-admin/src/pages/Customers/CustomerProfile.js [hunk 1]
```diff
@@ -313,6 +315,8 @@ const CustomerProfile = memo(() => {
     variables: { customerId }
   })
 
+  console.log(customerResponse)
```
Comment (josepfo) [inline new-lamassu-admin/src/pages/Customers/CustomerProfile.js]: console log

File: new-lamassu-admin/src/pages/Customers/components/EditableCard.js [hunk 2]
```diff
@@ -418,4 +418,87 @@ const EditableCard = ({
...
+const NonEditableCard = ({
+  fields,
+  hasImage,
+  state: _state,
+  title,
+  titleIcon
+}) => {
+  const classes = useStyles()
+
+  console.log('fields', fields)
```
Comment (josepfo) [inline new-lamassu-admin/src/pages/Customers/components/EditableCard.js]: console log

File: new-lamassu-admin/src/pages/Dashboard/Footer/Footer.js [hunk 3]
```diff
@@ -38,7 +37,7 @@ const Footer = () => {
   const withCommissions = R.path(['cryptoRates', 'withCommissions'])(data) ?? {}
   const classes = useStyles()
   const config = R.path(['config'])(data) ?? {}
-  const canExpand = R.keys(withCommissions).length > 4
+  // const canExpand = R.keys(withCommissions).length > 4
```
Comment (josepfo) [inline new-lamassu-admin/src/pages/Dashboard/Footer/Footer.js]: Is this intentional?

File: new-lamassu-admin/src/pages/Wizard/components/Wallet/Blockcypher.js [hunk 4]
```diff
@@ -55,6 +57,8 @@ const Blockcypher = ({ addData }) => {
 
   const accounts = data?.accounts ?? []
 
+  console.log(accounts)
```
Comment (josepfo) [inline new-lamassu-admin/src/pages/Wizard/components/Wallet/Blockcypher.js]: console log

**Review by josepfo — CHANGES_REQUESTED** [review]

**Comment by josepfo** [comment]
Has conflicts

File: new-lamassu-admin/src/pages/Dashboard/Footer/Footer.js (hunk 3 above)
Comment (chaotixkilla) [reply new-lamassu-admin/src/pages/Dashboard/Footer/Footer.js]: Variable hasn't been used for a long time, I'll just remove it

File: lib/routes/customerRoutes.js [hunk 5]
```diff
@@ -185,12 +186,30 @@ function sendSmsReceipt (req, res, next) {
...
+  if (_.isNil(customerId) || _.isNil(triggerId)) return next(httpError('Not Found', 404))
+
+  const settings = req.settings
+  const triggers = configManager.getTriggers(settings.config)
+  const trigger = _.find(it => it.id === triggerId)(triggers)
+
+  return externalCompliance.createApplicantAccessToken(settings, customerId, trigger.id)
+    .then(token => {
+      const url = `https://${process.env.NODE_ENV === 'production' ? `${process.env.HOSTNAME}` : `localhost:3001` }/${trigger.externalService}?customer=${customerId}&trigger=${trigger.id}&t=${token}`
+      process.env.NODE_ENV === 'production' && console.log(url)
```
Comment (josepfo) [inline lib/routes/customerRoutes.js]: What's this line's purpose?

**Review by josepfo — APPROVED** [review]

**Pull request closed without merging**